libs/api/domains/driving-license-book/src/drivingLicenseBook.service.ts

```diff
@@ -35,6 +35,7 @@ export class DrivingLicenseBookService {
     input: CreatePracticalDrivingLessonInput,
     user: User,
   ): Promise<Partial<Pick<PracticalDrivingLesson, 'id'>> | null> {
+    this.logger.debug(`driving-license-book: Create practical driving lesson`)
```

Comment:
The prefix is redundant, since the logger should log the context.
```suggestion
    this.logger.debug(`Create practical driving lesson`)
```